libcamera_app.hpp

```diff
@@ -785,4 +795,6 @@ class LibcameraApp
 	// For setting camera controls.
 	std::mutex control_mutex_;
 	ControlList controls_;
+	// Other:
+	uint64_t last_timestamp_ = 0;
 };
```

Comment:
`last_timestamp_`  probably wants to be initialised to in `StartCamera()` instead of here, as start/stop/start cycles will not use stale values.